src/components/PopoverWithMeasuredContent.js

```diff
@@ -30,6 +31,13 @@ const propTypes = {
     but in the case the children are not displayed, the measurement will not work. */
     measureContent: PropTypes.func.isRequired,
 
+    /** Static dimensions for the popover.
+     * Note: When passed, it will skip dimensions measuring of the popover, and provided dimensions will be used to calculate the anchor position. */
```

Comment:
```suggestion
    /** 
     * Static dimensions for the popover.
     * Note: When passed, it will skip dimensions measuring of the popover, and provided dimensions will be used to calculate the anchor position. 
     */
```